devtools/chain/specs/multi_nodes/chain-spec.toml

```
@@ -4,7 +4,7 @@
 
 [genesis]
 timestamp = 1680249207
-extra_data = []
+extra_data = { block_number = 0, hardforks = [] }
```

Move to the last field of `genesis` section, then:
```suggestion
[genesis.extra_data]
block_number = 0
hardforks = []
```